### ISSUE TITLE
请问一下train_MENet.py中下面这段代码为什么loss2、loss3、loss4不加Sm_loss1呢？

### ISSUE THREAD

**Issue by gl490402850** [yiwangtz/MENet#4]
请问一下train_MENet.py中下面这段代码为什么loss2、loss3、loss4不加Sm_loss1呢？
` loss1 = F.binary_cross_entropy_with_logits(out1, mask) + iou_loss(out1, mask) + structure_loss(out1, mask) + pr_loss(out1, mask) + Sm_loss1(out1,mask)
            loss2 = F.binary_cross_entropy_with_logits(out2, mask) + iou_loss(out2, mask) + structure_loss(out2, mask) + pr_loss(out2, mask) #+ Sm_loss1(out2,mask)
            loss3 = F.binary_cross_entropy_with_logits(out3, mask) + iou_loss(out3, mask) + structure_loss(out3, mask) + pr_loss(out3, mask) #+ Sm_loss1(out3,mask)
            loss4 = F.binary_cross_entropy_with_logits(out4, mask) + iou_loss(out4, mask) + structure_loss(out4, mask) + pr_loss(out4, mask)  # + Sm_loss1(out4,mask)`

**Comment by Gi-gigi** [comment]
您好，我在这里追问一下。请问：1.为什么按照原始代码训练测试出来的结果不一致呢？2.您的batchsize设置的是多少呢？

能否说明一下？
